app/controllers/users_controller.rb

```diff
@@ -63,12 +63,12 @@ def profile
 
   def resource_params
     params.fetch(:user, {}).permit(
-        :username, :email, :phone, :password,
-        fname_attributes: [:ip, :rp, :dp, :vp, :tp, :pp],
-        iname_attributes: [:ip, :rp, :dp, :vp, :tp, :pp],
-        oname_attributes: [:ip, :rp, :dp, :vp, :tp, :pp],
-        positions_attributes: [:id, :appointment_id, :acl_position_role,
-                               :acl_position_department, :started_at, :primary, :_destroy]
+      :username, :email, :phone, :password,
+      fname_attributes: [:ip, :rp, :dp, :vp, :tp, :pp],
+      iname_attributes: [:ip, :rp, :dp, :vp, :tp, :pp],
+      oname_attributes: [:ip, :rp, :dp, :vp, :tp, :pp],
+      positions_attributes: [:id, :appointment_id, :acl_position_role,
+                            :acl_position_department, :started_at, :primary, :_destroy]
```

Comment:
Line is too long. [87/80]<br>Align the elements of an array literal if they span more than one line.